components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/main/resources/APIResources.yaml

```
@@ -629,7 +721,7 @@ components:
           example: before
         href:
           type: string
-          example: /o/orgName/api-resources?after=NDoy
+          example: /t/carbon.super/api/server/v1/api-resources?after=NDoy
```

```suggestion
          example: /api/server/v1/api-resources?after=NDoy
```
/t/carbon.super/ will not return now. /t/{tennat-domain} will be returned in other tenants. For this example we don't need to add it